src/Servers/Kestrel/Core/src/Http2Limits.cs

```diff
@@ -81,7 +81,7 @@ public int MaxFrameSize
         /// <summary>
         /// Indicates the size of the maximum allowed size of a request header field sequence. This limit applies to both name and value sequences in their compressed and uncompressed representations.
         /// <para>
-        /// Value must be greater than 0, defaults to 8192
+        /// Value must be greater than 0, defaults to 2^14 (16,384)
```

Comment:
Nit: I think it's OK just to use 16,384. People who care whether or not it's a power of two will intuitively know that it is.